pymc3/distributions/continuous.py

```diff
@@ -4026,3 +4045,204 @@ def logcdf(value, mu, sigma):
+
+    where :math:`d_k = 2(k - 0.5)^2\pi^2 + z^2/2`, :math:`Ga(h, 1)` is a gamma
+    random variable with shape  parameter ``h`` and scale parameter ``1``.
+
+    .. plot::
+
+        import matplotlib.pyplot as plt
+        import numpy as np
+        from polyagamma import polyagamma_pdf
+        plt.style.use('seaborn-darkgrid')
+        x = np.linspace(0.01, 5, 500);x.sort()
+        hs = [1., 5., 10., 15.]
+        zs = [0.] * 4
+        for h, z in zip(hs, zs):
+            pdf = polyagamma_pdf(x, h=h, z=z)
+            plt.plot(x, pdf, label=r'$h$ = {}, $z$ = {}'.format(h, z))
+        plt.xlabel('x', fontsize=12)
+        plt.ylabel('f(x)', fontsize=12)
+        plt.legend(loc=1)
+        plt.show()
+
+    ========  =============================
+    Support   :math:`x \in (0, \infty)`
+    Mean      :math:`dfrac{h}{4} if :math:`z=0`, :math:`\dfrac{tanh(z/2)h}{2z}` otherwise.
+    Variance  :math:`0.041666688h` if :math:`z=0`, :math:`\dfrac{h(sinh(z) - z)(1 - tanh^2(z/2))}{4z^3}` otherwise.
+    ========  =============================
+
+    Parameters
+    ----------
+    h: float, optional
+        The shape parameter of the distribution (h > 0).
+    z: float, optional
+        The exponential tilting parameter of the distribution.
+
+    Examples
+    --------
+    .. code-block:: python
+
+        rng = np.random.default_rng()
+        with pm.Model():
+            x = pm.PolyaGamma('x', h=1, z=5.5)
+        with pm.Model():
+            x = pm.PolyaGamma('x', h=25, z=-2.3, rng=rng, size=(100, 5))
+
+    References
+    ----------
+    .. [1] Polson, Nicholas G., James G. Scott, and Jesse Windle.
+           "Bayesian inference for logistic models using Pólya–Gamma latent
+           variables." Journal of the American statistical Association
+           108.504 (2013): 1339-1349.
+    .. [2] Windle, Jesse, Nicholas G. Polson, and James G. Scott.
+           "Sampling Polya-Gamma random variates: alternate and approximate
+           techniques." arXiv preprint arXiv:1405.0506 (2014)
+    .. [3] Luc Devroye. "On exact simulation algorithms for some distributions
+           related to Jacobi theta functions." Statistics & Probability Letters,
+           Volume 79, Issue 21, (2009): 2251-2259.
+    .. [4] Windle, J. (2013). Forecasting high-dimensional, time-varying
+           variance-covariance matrices with high-frequency data and sampling
+           Pólya-Gamma random variates for posterior distributions derived
+           from logistic likelihoods.(PhD thesis). Retrieved from
+           http://hdl.handle.net/2152/21842
+    """
+    rv_op = polyagamma
+
+    @classmethod
+    def dist(cls, h=1.0, z=0.0, **kwargs):
+        hh = at.as_tensor_variable(floatX(h))
+        zz = at.as_tensor_variable(floatX(z))
+
+        msg = f"The variable {hh} specified for PolyaGamma has non-positive "
+        msg += "values, making it unsuitable for this parameter."
+        Assert(msg)(hh, at.all(at.gt(hh, 0.00001)))
```

Comment:
Why not just `gt(hh, 0)`?

Reply:
Because of this: https://github.com/zoj613/polyagamma/blob/20b8e754550844ebf11821e1984bb5611e453816/polyagamma/_polyagamma.pyx#L234

At the time of implementation, I figured there is no sense in letting the shape parameter any smaller than `1e-04` since there isn't any practical case where that is useful. I figured i'd leave it like that until a user requires it to be arbitrarily small.

Reply:
Relaxed it in [`1174daf` (#4531)](https://github.com/pymc-devs/pymc3/pull/4531/commits/1174dafd5621584f6b73a7b2b93f9544c5d1da4b)